### ISSUE TITLE
Metadata for topdown view

### ISSUE THREAD

**Issue by Tushar-N** [allenai/ai2thor#353]
I'm trying to get other metadata (depth, segmentation etc.) for the topdown view for environments that matches with the frame information, but it looks like there's a mismatch between the two. 
For example, in `FloorPlan1`
| image  | instance segmentation |
| ------------- | ------------- |
| ![topdown](https://user-images.githubusercontent.com/4995097/74206810-a18ac480-4c42-11ea-9a80-bef8f753fd87.png)  | ![segm](https://user-images.githubusercontent.com/4995097/74206814-a3ed1e80-4c42-11ea-88cb-39d152287d86.png)  |

Looks like different cameras generate information for the two views. I know this functionality isn't officially supported, but I was wondering if there's a simple fix for this. Thanks.

**Comment by winthos** [comment]
Yes, this is not officially supported, and it seems the behavior you are seeing is due to the depth/segmentation cameras being defaulted to "Perspective" mode. When using the `ToggleMapView` action, the main camera changes to "Orthographic" mode which creates the flat image you are seeing on the left.  

There is not a straightforward way to toggle the depth/segmentation/ etc. cameras to also switch to Orthographic mode. If they are switched to orthographic mode manually, certain metadata, like depth, would also stop working. The depth frame relies on the Perspective mode camera to, well, have the perspective to generate depth. In Orthographic mode, all perspective is removed so the depth frame would have no gradient.

**Comment by Tushar-N** [comment]
Got it, thanks! For reference, I was hoping to get an occupancy map (using depth) and a semantic map with object classes (using the instance segmentations) in the orthographic view, but sounds like this isn't possible.

**Comment by winthos** [comment]
For instance segmentation, it should accurately mask object instances regardless of perspective/orthographic mode. The only issue that might occur is in orthographic mode, some objects might not be visible in the same way, or might be obscured by object above them (ie: a mug below a set of cabinets would be obscured by the cabinets).
